app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -2002,5 +2014,75 @@ public void onNewMessages(Message[] messages) {
         if (narrowedList != null) {
             narrowedList.onNewMessages(messages);
         }
+        if (!currentList.scrolledToLastMessage())
+            showSnackbarNotification(messages); //Show notification
+    }
+
+    Snackbar snackbar;
+    CoordinatorLayout.LayoutParams snackBarParams;
+    private void setupSnackBar() {
+        final CoordinatorLayout coordinatorLayout = (CoordinatorLayout) findViewById(R.id.coordinatorLayout);
+        snackbar = Snackbar.make(coordinatorLayout, "", Snackbar.LENGTH_LONG);
+        View v = snackbar.getView();
+        snackBarParams = (CoordinatorLayout.LayoutParams) v.getLayoutParams();
+        snackBarParams.gravity = Gravity.TOP;
+        v.setLayoutParams(snackBarParams);
+        TypedValue tv = new TypedValue();
+        if (getTheme().resolveAttribute(android.R.attr.actionBarSize, tv, true))
+            mToolbarHeightInPx = TypedValue.complexToDimensionPixelSize(tv.data, getResources().getDisplayMetrics());
+    }
+
+    NarrowFilter narrowFilter;
+
+    private void showSnackbarNotification(Message[] messages) {
+        MutedTopics mutedTopics = MutedTopics.get();
```

Comment:
Hmmm, yeah this is intentional what could be the possible solution be? 
Let the snackbar disappear and display the snackbar again?